gnocchi/setuptools.py

```diff
@@ -0,0 +1,123 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import absolute_import
+
+import os
+import subprocess
+
+from distutils import version
+from setuptools.command import develop
+from setuptools.command import easy_install
+from setuptools.command import egg_info
+from setuptools.command import install_scripts
+
+
+SCRIPT_TMPL = """
```

Comment:
do we really need a template for this? Can't we just ship this as a script?

Reply:
The current script have a wrong header (we hardcode python2), I use a tmpl to add the sheban with the correct python version, I ask setuptools/easy_install what the binary to use at install time, so the script will always work. Also we can't use an entry_point due to the wsgi stuff.